packages/commonwealth/client/scripts/state/api/proposals/fetchCompletedCosmosProposals.ts

```diff
@@ -14,16 +16,16 @@ interface CompletedProposalsProps {
 const useCompletedCosmosProposalsQuery = ({
   isApiReady,
 }: CompletedProposalsProps) => {
-  const cosmosChain = app.chain as Cosmos;
   return useQuery({
     queryKey: ['completedProposals', { chain: app.activeChainId() }],
     queryFn: fetchCompletedProposals,
-    enabled: isApiReady,
+    enabled: app.chain?.base === ChainBase.CosmosSDK && isApiReady,
     retry: 3,
     // completed proposals never change, so we can cache
     // the old ones forever. React Query will load new
-    // ones in the background.
+    // ones in the background, and update if needed.
     cacheTime: Infinity,
+    staleTime: 1000 * 15, // 15sec - don't need constant pinging here
```

Comment:
the pattern we have is that stale time is in the separate const.

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/75a962d7-5d4e-4970-b70f-57e453b0d2b4)

Reply:
done